ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/secureboot/confidential-computing/provision/cucushift-installer-rehearse-gcp-ipi-secureboot-confidential-computing-provision-chain.yaml

```diff
@@ -9,6 +9,9 @@ chain:
     - ref: ipi-conf-gcp-secureboot
     - chain: ipi-install
     - ref: enable-qe-catalogsource
+    - ref: cucushift-installer-check-gcp-secureboot
+    - ref: cucushift-installer-check-gcp-machine-type
```

Comment:
Do you still remember why we insert `ipi-conf-gcp-control-plane-node-type` in this install workflow?

Reply:
It's part of post-installation check of Confidential Computing, which demands certain instance types, including N2D and C2D.

Reply:
Sorry, the ipi-conf-gcp supports COMPUTE_NODE_TYPE only, so we need the step for specifying control-plane's instance type.

Reply:
Got it, so the root reason is `Confidential Computing` only can be run on certain instance types.

Reply:
Yes.